src/Monolog/Processor/UidProcessor.php

```diff
@@ -43,4 +45,9 @@ public function getUid(): string
     {
         return $this->uid;
     }
+
+    public function restart(int $length = 7)
+    {
+        $this->__construct($length);
```

Comment:
I would rather move the uid generation to a private method and call it, rather than reusing `__construct` (which will be necessary anyway when applying the suggestions). I don't want to push people to think that calling `__construct` as a method is a supported usage of the library.